src/Native/Runtime/GCHelpers.cpp

```diff
@@ -25,6 +26,8 @@
 #include "threadstore.inl"
 #include "thread.inl"
 
+extern uint64_t dead_threads_non_alloc_bytes;
```

Comment:
It may look better to put this inside `RedhawkGCInterface`. E.g. similar to how `tls_pLastAllocationEEType` is done